pkg/spanconfig/spanconfigjob/job.go

```diff
@@ -40,35 +39,34 @@ var _ jobs.Resumer = (*resumer)(nil)
 // Resume implements the jobs.Resumer interface.
 func (r *resumer) Resume(ctx context.Context, execCtxI interface{}) error {
 	execCtx := execCtxI.(sql.JobExecContext)
+	rc := execCtx.ConfigReconciliationJobDeps()
 
-	// TODO(zcfgs-pod): Listen in on rangefeeds over system.{descriptor,zones}
-	// and construct the right update, instead of this placeholder write over
-	// the same keyspan over and over again.
-	// TODO(zcfgs-pod): How do we test this?
-	nameSpaceTableStart := execCtx.ExecCfg().Codec.TablePrefix(keys.NamespaceTableID)
-	nameSpaceTableSpan := roachpb.Span{
-		Key:    nameSpaceTableStart,
-		EndKey: nameSpaceTableStart.PrefixEnd(),
+	updates, err := rc.Watch(ctx)
```

Comment:
Name this as updateCh. This error is terminal, if there's an error establishing these rangefeeds, the job will never work. Should we keep retrying until success? 

Should the dependency be a pre-constructed rangefeed (which if we encountered an error, we'd have done so early during start-up/init). I guess that doesn't work if the job gets bounced around between nodes, I guess we want to establish a rangefeed on Resume each time. 

I'm not sure how we should be dealing with this error, but simply logging it here and going forward feels off.